articles/healthcare-apis/fhir/convert-data.md

```diff
@@ -85,7 +85,7 @@ $convert-data takes a [Parameter](http://hl7.org/fhir/parameters.html) resource
         ...
         ...
       "request": {
-        "method": "PUT",
+        "method": "POST",
```

Comment:
This is part of the sample response bundle that you get from $convert-data. This is not a request sample.